### ISSUE TITLE
helm_release completes successfully but nothing is deployed

### ISSUE THREAD

**Issue by fl-max** [hashicorp/terraform-provider-helm#404]
### Terraform Version

$ terraform -v
Terraform v0.12.18
+ provider.helm v1.0.0
+ provider.kubernetes v1.11.0
+ provider.random v2.2.1

### Affected Resource(s)

- helm_release

### Terraform Configuration Files
```hcl
provider "helm" {
  version    = ">= 1.0.0"
}

resource "random_pet" "release-name" {
  length = 2
  separator = "-"
}

resource "helm_release" "my-chart" {
  name          = random_pet.release-name.id
  chart         = "../my-chart"
  version       = var.chart_version
  namespace     = "my-chart-ns"
  recreate_pods = false

  values = [
    file("helm.yaml")
  ]
  timeout = 900
}
```

### Expected Behavior

```bash
...
helm_release.my-chart: Still creating... [1m40s elapsed]
helm_release.my-chart: Creation complete after 1m48s [id=smiling-frog]
```

```bash
$ kubectl -n my-chart-ns get pods
# List of Pods here
```

### Actual Behavior

```bash
...
helm_release.my-chart: Still creating... [1m40s elapsed]
helm_release.my-chart: Creation complete after 1m48s [id=smiling-frog]
```

```bash
$ kubectl -n my-chart-ns get pods
No resources found in my-chart-ns namespace.
```

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

When trying to deploy the same chart from `helm`, I received this error:
```bash
Error: create: failed to create: Request entity too large: limit is 3145728
```

I was trying to test `my-chart` from the same directory as the Chart; putting all the TF code in a sub-dir `example/` which also contained the `.terraform/` state/binaries folder. Removing this folder and keep the chart directory cleaned resolved the issue.

Opening this as an issue as the Provider should bubble up the error.

**Comment by pierresteiner** [comment]
Looks like a duplicate of https://github.com/terraform-providers/terraform-provider-helm/issues/401

**Comment by fl-max** [comment]
Hmm, @pierresteiner thought Helm3 doesn't automatically create the namespace anymore? It's possible they could be duplicate but @mrkwtz hasn't given any further details or debugged with helm cli.

**Comment by jrhouston** [comment]
This was resolved in 1.1.0.
